diff --git a/libs/libutils/src/pgsqlversions.cpp b/libs/libutils/src/pgsqlversions.cpp
index 726e713c8..ccf15d6e5 100644
--- a/libs/libutils/src/pgsqlversions.cpp
+++ b/libs/libutils/src/pgsqlversions.cpp
@@ -29,12 +29,13 @@ namespace PgSqlVersions {
 	PgSqlVersion150("15.0"),
 	PgSqlVersion160("16.0"),
 	PgSqlVersion170("17.0"),
-	DefaulVersion = PgSqlVersion170,
+	PgSqlVersion180("18.0"),
+	DefaulVersion = PgSqlVersion180,
 	MinimumVersion = PgSqlVersion100;
 
 	const QStringList
 	AllVersions = {
-		PgSqlVersion170, PgSqlVersion160,
+		PgSqlVersion180, PgSqlVersion170, PgSqlVersion160,
 		PgSqlVersion150, PgSqlVersion140,
 		PgSqlVersion130, PgSqlVersion120,
 		PgSqlVersion110, PgSqlVersion100
diff --git a/libs/libutils/src/pgsqlversions.h b/libs/libutils/src/pgsqlversions.h
index 58101edb8..0737684dd 100644
--- a/libs/libutils/src/pgsqlversions.h
+++ b/libs/libutils/src/pgsqlversions.h
@@ -39,6 +39,7 @@ namespace PgSqlVersions {
 	PgSqlVersion150,
 	PgSqlVersion160,
 	PgSqlVersion170,
+	PgSqlVersion180,
 	MinimumVersion,
 	DefaulVersion;
 
